packages/govuk-frontend/tasks/build/package.unit.test.mjs

```diff
@@ -151,6 +151,7 @@ describe('packages/govuk-frontend/dist/', () => {
       'govuk-prototype-kit.config.json',
       'gulpfile.mjs',
       'package.json',
+      'package.json.unit.test.mjs',
```

Comment:
**comment** Something for another PR, but there are a couple of `.mjs` files that end up in the final package when they shouldn't (see the couple of `.test.mjs` and `.config.mjs` files after this line). There's a `*.test.*` entry in `.npmignore`, but it doesn't seem to do much.

Reply:
**note** We only publish a subset of what's in `package/govuk-frontend` to npm because we're filtering it using the [`files` field](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files) of `packages/govuk-frontend/package.json`:

https://github.com/alphagov/govuk-frontend/blob/f04749907cbab1fddc3dc488b42319c7022ca1a2/packages/govuk-frontend/package.json#L8-L13

Agree it's a little confusing as we have a combination of exclusions at build time, exclusions using `.npmignore` and a list of inclusions in that `files` field that all intersect with each other, but I think what we're publishing is broadly 'correct'.